charts/ecosystem/templates/custom-resource-monitor.yaml

```diff
@@ -0,0 +1,99 @@
+#
+# Copyright contributors to the Galasa project
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Release.Name }}-custom-resource-monitor
+  labels:
+    galasa-monitor: custom
+    app: {{ .Release.Name }}-custom-resource-monitor
+spec:
+  replicas: 0
+  strategy:
+    type: Recreate
+  selector:
+    matchLabels:
+      app: {{ .Release.Name }}-custom-resource-monitor
+  template:
+    metadata:
+      name: {{ .Release.Name }}-custom-resource-monitor
+      labels:
+        app: {{ .Release.Name }}-custom-resource-monitor
+    spec:
+      serviceAccountName: galasa
+      nodeSelector:
+        kubernetes.io/arch: {{ .Values.architecture }}
+        {{- if .Values.nodeSelectors }}
+{{ toYaml .Values.nodeSelectors | indent 8 }}
+        {{- end }}
+      initContainers:
+        - name: wait-for-api
+          image: {{ .Values.kubectlImage }}
+          imagePullPolicy: {{ .Values.pullPolicy }}
+          command:
+            - kubectl
+          args:
+            - wait
+            - pods
+            - -l=app={{ .Release.Name }}-api
+            - --for=condition=Ready
+            - --timeout=180s
+      containers:
+      - name: resource-monitor
+        image: {{ .Values.galasaRegistry }}/{{ .Values.galasaBootImage }}:{{ .Values.galasaVersion }}
+        imagePullPolicy: {{ .Values.pullPolicy }}
+        command: ["java"]
+        args: 
+        - -jar
+        - boot.jar
+        - --obr
+        - file:galasa.obr
+        - --resourcemanagement
+        - --bootstrap
+        - http://{{ .Release.Name }}-api:8080/bootstrap
+        env:
+        - name: NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
+        - name: GALASA_CONFIG_STORE
+          value: etcd:{{ include "cps.url" . }}
+        - name: GALASA_DYNAMICSTATUS_STORE
+          value: etcd:{{ include "dss.url" . }}
+        - name: GALASA_RESULTARCHIVE_STORE
+          value: couchdb:http://{{ .Release.Name }}-couchdb:5984
+        - name: GALASA_CREDENTIALS_STORE
+          value: etcd:{{ include "creds.url" . }}
+        - name: GALASA_RAS_TOKEN
+          valueFrom:
+            secretKeyRef:
+              name: {{ .Release.Name }}-couchdb-secret
+              key: GALASA_RAS_TOKEN
+        {{- if .Values.cleanupMonitor.stream }}
+        - name: GALASA_CLEANUP_MONITOR_STREAM
+          value: "{{ .Values.cleanupMonitor.stream }}"
+        {{- end }}
+        - name: GALASA_MONITOR_INCLUDES_REGEXES
+          value: "{{ join "," .Values.cleanupMonitor.includes }}"
+        - name: GALASA_MONITOR_EXCLUDES_REGEXES
```

Comment:
GLOB_PATTERNS?

Reply:
Addressed in https://github.com/galasa-dev/helm/pull/73/commits/6c4cebafb1f57664ea78dd16df11f913c4328d71